packages/js/github-actions/actions/eslint-annotation/action.yml

```diff
@@ -0,0 +1,18 @@
+name: ESlint annotattion formatter
+description: Annotate eslint results via eslint formatter.
+
+inputs:
+  formatter-dest:
+    description: The relative path to place the eslint formatter script. Please use a destination file path within the caller repository since the script will depends on the eslint package installed in it.
+    default: "./eslintFormatter.cjs"
+
+runs:
+  using: composite
+  steps:
+    # Copy formatter script to the destination file path.
+    - shell: bash
+      run: |
+        SCRIPT_DEST="${{ inputs.formatter-dest }}"
+        mkdir -p $(dirname "$SCRIPT_DEST")
+        echo '/* eslint-disable */' > "$SCRIPT_DEST"
+        cat "${{ github.action_path }}/eslintFormatter.cjs" >> "$SCRIPT_DEST"
```

Comment:
In my mind would be cool to just load the action and forgot about anything else.

Reply:
The way `eslint` is performed can vary slightly between repos, so this action only provides the formatter for the callers to use. In addition, usually a caller repo still sets up `eslint` configuration locally, it eventually needs to know how to run `eslint` itself.

If this action performs `eslint`, it will need to add `eslint` options to the inputs of this action if want to retain the flexibility, so I think it would be better to provide only formatter.